app/src/main/java/com/example/datarsd1/icebreaker/QuizActivity.java

```diff
@@ -0,0 +1,207 @@
+            else
+                {b4.setBackgroundResource(R.drawable.buttonshape); b4.setTextColor(getResources().getColor(R.color.textColor));}
+        }
+
+        b1.setText(q.getOptions()[0]);
+        b2.setText(q.getOptions()[1]);
+        b3.setText(q.getOptions()[2]);
+        b4.setText(q.getOptions()[3]);
+
+        int profile = getResources().getIdentifier(answer.toLowerCase().replaceAll("\\s+", ""), "drawable", getPackageName());
+        ImageView pic = (ImageView) findViewById(R.id.imageView);
+        pic.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent contactIntent = new Intent(ContactsContract.Intents.Insert.ACTION);
+                contactIntent.setType(ContactsContract.RawContacts.CONTENT_TYPE);
+                contactIntent.putExtra(
+                        ContactsContract.Intents.Insert.NAME, answer
+                );
+                startActivity(contactIntent);
+            }
+        });
+        pic.setImageResource(profile);
+
+        timer.start();
+    }
+
+    public void onClick(View v)
+    {
+        timer.cancel();
+        score++;
+        switch (v.getId())
+        {
+            case R.id.option1:
+                if (!(answer.equalsIgnoreCase(b1.getText().toString())))
+                {
+                    score--;
+                    Toast.makeText(getApplicationContext(), "The correct answer was " + answer, Toast.LENGTH_LONG).show();
+                }
+                askQuestion();
+                break;
+            case R.id.option2:
+                if (!(answer.equalsIgnoreCase(b2.getText().toString())))
+                {
+                    score--;
+                    Toast.makeText(getApplicationContext(), "The correct answer was " + answer, Toast.LENGTH_LONG).show();
+                }
+                askQuestion();
+                break;
+            case R.id.option3:
+                if (!(answer.equalsIgnoreCase(b3.getText().toString())))
+                {
+                    score--;
+                    Toast.makeText(getApplicationContext(), "The correct answer was " + answer, Toast.LENGTH_LONG).show();
+                }
+                askQuestion();
+                break;
+            case R.id.option4:
+                if (!(answer.equalsIgnoreCase(b4.getText().toString())))
+                {
+                    score--;
+                    Toast.makeText(getApplicationContext(), "The correct answer was " + answer, Toast.LENGTH_LONG).show();
+                }
+                askQuestion();
+                break;
+            case R.id.returnButton:
+                score = 0;
+                timer.cancel();
+                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
+                startActivity(intent);
+                Log.d("hi","hi");
+                break;
+        }
+    }
+
+    protected void onPause()
```

Comment:
NEVER call onPause without onResume! Remember that onCreate only calls when the activity is initialized, so coming back to the screen kills the timer!